marklogic-data-hub-central/ui/src/pages/EntityTypes.tsx

```diff
@@ -43,18 +47,92 @@ const EntityTypes: React.FC = () => {
           }
 
     }
+
+  //GET all the flow artifacts
+  const getFlows = async () => {
+        try {
+            let response = await axios.get('/api/flows');
+            if (response.status === 200) {
+                setFlows(response.data);
+                console.log('GET flows successful', response);
```

Comment:
Can we please get rid of all the console.log and only have console.error where applicable.